tests/func/test_data_cloud.py

```diff
@@ -149,6 +152,18 @@ def get_ssh_url():
     )
 
 
+def get_ssh_url_mocked(user, port):
+    if os.name == "nt":
+        url = "ssh://{}@127.0.0.1:{}:{}".format(
```

Comment:
I don't think this second `:` is necessary. Should it be something like
```
path = get_local_storagepath()
if os.name == "nt":
    path = "/" + path.replace("\\", "/")
url = "ssh://{}@127.0.0.1:{}{}".format(user, port, path)
``` 
? Also please don't forget to add a note here on why you are doing that `replace()` magic.

Reply:
No, It will raise an exception: ` DvcException: couldn't get the '/c:/users/cao/appdata/local/temp/dvc-test.7968.outdjx.e8373c98-b117-479c-9ed3-000a974182da' remote directory files list` . Because the initial slash `/` before `c:/`. That is the reason I provide this indicator to solve in Windows. Is that make sense or do you have other solution?

Reply:
> Also please don't forget to add a note here on why you are doing that replace() magic.

I will append the note